### PR TITLE
Chapter 9: Advanced login

### PR DESCRIPTION
**Pull request by stephanie1997-kek**

### CODE REVIEW

**Comment by stephanie1997-kek** [comment]
ready

**Comment by stephanie1997-kek** [comment]
Ready.

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -1,14 +1,16 @@
 <header class="navbar navbar-fixed-top navbar-inverse">
   <div class="container">
-    <%= link_to "sample app", "#", id: "logo" %>
+    <%= link_to t(".logo"), root_path, id: "logo" %>
```
Comment (chungpv-1008) [inline app/views/layouts/_header.html.erb]:
```suggestion
    <%= link_to t(".logo"), root_path, id: "logo" %>
```

**Review by chungpv-1008 — COMMENTED** [review]
please fix comment, then merge the pull request yourself

File: app/views/layouts/_header.html.erb (hunk 1 above)
Comment (stephanie1997-kek) [reply app/views/layouts/_header.html.erb]: done

**Comment by vanvtt-0952** [comment]
merged

**Pull request merged**